src/helpers.ts

```diff
@@ -13,11 +14,11 @@ export async function withTempDir<T>(fn: (dir: string) => Promise<T>) {
     result = await fn(dir);
   } catch (err) {
     d('work failed');
-    await fs.remove(dir);
+    await fs.rm(dir, { recursive: true });
     throw err;
   }
   d('work succeeded');
-  await fs.remove(dir);
+  await fs.rm(dir, { recursive: true });
```

Comment:
Just checking my assumptions here, but we had to add the recursive option here due to a behavior change when we moved from `fs-extra` to Node's `fs`?

Reply:
Yup. It's not very explicitly documented but `fs-extra`'s remove is recursive while Node's `fs` is not by default.

If we look at [the latest `fs-extra` code for `remove`](https://github.com/jprichardson/node-fs-extra/blob/d96f2655f181f1c6b1641711619400140a89d027/lib/remove/index.js#L6-L8) we can see it's just calling through to `fs.rm(path, { recursive: true, force: true })` (it's technically using `graceful-fs`, but looks like that library just passes through `fs.rm` as-is - yay layers of abstraction).

But this is a good callout, because I think to ensure there's no regression in behavior here we need to add the `force: true` option. Adding suggestions to do so.

Reply:
Keeping `force: true` makes things easier for migration purposes, but might just be masking an erroneous call if some folder never exists?

For example, I was running into a case in `@electron/get` where we were always attempting to delete a folder path that didn't exist.

Reply:
Fair point. I think for migration purposes we should strive for no intentional behavior change, and then we can follow-up with that change after?

Reply:
Yep, I think removing `force` calls after the fact could be a `good-first-issue` since the acceptance criteria is pretty clear!